src/tests/ftest/pool/evict_test.py

```diff
@@ -175,7 +175,8 @@ def test_evict(self):
         The handle is removed.
         The test verifies that the other two pools were not affected
         by the evict
-        :avocado: tags=all,pool,pr,full_regression,small,poolevict,DAOS_5610
+        :avocado: tags=all,pool,pr,daily_regression,full_regression,small,poolevict
```

Comment:
(pylint-line-too-long) Line too long (83/80)

Reply:
This should be fixed.

Reply:
Done.

Reply:
Done.

Reply:
Done.